### DISCUSSION TITLE
__init__.py contains syntax error, yet another typo

### DISCUSSION THREAD

**Discussion by infradom** [wills106/homeassistant-solax-modbus#65] [General]
The latest main line does not compile around line 526 - 580
I also saw that a previous correction has disappeared in const.py: should be:
   HYB_GROUP_BITS = 0x0C00 # hybrid or AC flags

**Comment by infradom** [comment]
created pull request #66 to re-correct the HYB-GROUP-BITS typo
Due to this typo, the PV entities were still visible on an AC model

**Reply by wills106** [reply]
[Release 0.5.0b9b](https://github.com/wills106/homsassistant-solax-modbus/releases/tag/0.5.0b9b) should fix this.
